pkg/query-service/app/metrics/v4/query_builder.go

```diff
@@ -3,8 +3,11 @@ package v4
 import (
 	"fmt"
 	"strings"
+	"time"
 
+	metricsV3 "go.signoz.io/signoz/pkg/query-service/app/metrics/v3"
```

Comment:
Should we import just the `options` from `metrics/v3` or create a separate options for `v4`?

Reply:
This was required to make both v3 and v4 co-exist.

Reply:
ok